agones/README.md

```diff
@@ -55,6 +44,68 @@ terraform apply -var project="<YOUR_GCP_ProjectID>"
 gcloud container clusters get-credentials --zone us-west1-c agones
 ```
 
+## Running Tests
+
+To run the Agones integration tests with a Quilkin image, you will need to specify the image 
+to be used along with the `cargo test` command and ensure that is available on the currently authenticated Kubernetes 
+cluster.
+
+This can be done through the `IMAGE_TAG` environment variable like so:
+
+```shell
+IMAGE_TAG=us-docker.pkg.dev/my-project-name/dev/quilkin:0.4.0-auyz cargo test
+```
+
+### Build, Push and Test in one Go 💪
+
+The [`build/Makefile`](../build/Makefile) provides a targets that can be executed to build a development image, 
+push it a appropriate location, and run the set of Agones integration tests, depending on where you Kubernetes 
+cluster is set up. 
+
+#### Minikube
+
+This target assumes that you have a [working minikube cluster](#creating-an-agones-minikube-cluster),
+under the profile `quilkin`, with Agones installed, and the local `.kube` configuration is currently
```

Comment:
In the [Agones minikube documentation](https://agones.dev/site/docs/installation/creating-cluster/minikube/), it directs you to use a profile named `agones`. 

I went in the direction to push people to keep the Quilkin testing separate from other things, so picked a different default profile for these targets -- but what do you think, should it be `agones` ?

Can also change it later if it starts to annoy people.